README.md

```diff
@@ -94,7 +94,7 @@ contract MyContract is ILockCallback {
 
 ## License
 
-The primary license for Uniswap V4 Core is the Business Source License 1.1 (`BUSL-1.1`), see [LICENSE](https://github.com/Uniswap/v4-core/blob/main/LICENSE). Minus the following exceptions:
+The primary license for Uniswap V4 Core is the GNU Affero General Public License v3.0 (`AGPL-3.0`), see [LICENSE](./LICENSE). Minus the following exceptions:
```

Comment:
per ChatGPT Esquire:

The statement "Minus the following exceptions" in the README.md file is a bit unclear. It might be better to explicitly state what these exceptions are and how they relate to the new license. For example, it could be clarified whether these exceptions are also relicensed under the AGPL-3.0 or if they remain under their original licenses.

Reply:
hey ChatGPT :-D - I think this part is clear imho since they state for each exception which open-source license is applied :)